node/node_test.go

```diff
@@ -3447,6 +3447,51 @@ func TestNode_GetAccountAccountExistsShouldReturn(t *testing.T) {
 	require.Equal(t, testscommon.TestAddressAlice, recovAccnt.OwnerAddress)
 }
 
+func TestNode_GetAccountAccountWithKeysErrorShouldErr(t *testing.T) {
```

Comment:
Is this test name ok?

Reply:
rename to TestNode_GetAccountAccountWithKeysErrorShouldFail